io-ballerina/src/io/character/file_string_io.bal

```diff
@@ -14,12 +14,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
-# Read the entire file content as a string.
+# Read the entire file content as a string
```

Comment:
Reads?

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/d79aba6075b65a53328cf0c0b830ae4aa5f4ba7d